README.md

```diff
@@ -1,5 +1 @@
-# ![](logo.png)
-
-talk.manageiq.org has been replaced by [GitHub Discussions](https://github.com/ManageIQ/manageiq/discussions). Please start any new discussions there.
-
-To see the old content, [click here](https://web.archive.org/web/https://talk.manageiq.org) or follow an old link and you will be redirected to the page on the internet archive.
+# talk.manageiq.org
```

Comment:
We might want to keep the old README content for browsers of the repo.  If we have an index.md, then GitHub won't render the README.md as the index.html.